foosball/table/main.fbp

```diff
@@ -30,7 +30,7 @@ Max(GoalsMax)
 
 ResetButton(MainResetButton)
 
-ScoreBoard(score-counter/client-score-counter:device_id="666a3d6a9d194a23b90a24573558d2f4")
+ScoreBoard(score-counter/client-score-counter:device_id="580a3d6a9d194a23b90a24573558d2f4")
```

Comment:
Is there any reason to change the device id?
Anyway, if its required for any reason (what I hope isn't the case), you should update the user instructions (Readme), otherwise people trying to use this demo won't be able to make these two boards talk to each other.